src/app/main/component/shared/components/input-error/input-error.component.ts

```diff
@@ -48,7 +48,7 @@ export class InputErrorComponent implements OnInit {
   }
 
   getMinlengthErrorMessage(minlength: number): string {
-    return minlength === 27 ? this.validationErrors.minlengthDescription : this.validationErrors.minlength;
+    return minlength === 20 ? this.validationErrors.minlengthDescription : this.validationErrors.minlength;
```

Comment:
Maybe create for this a separate constant

Reply:
Thanks! I've fixed this issue a bit different and I hope better way